package.json

```diff
@@ -187,7 +187,8 @@
     "packages/*/build/**",
     "packages/*/src/lib/**",
     "coverage/**",
-    "external/**"
+    "external/**",
+    "packages/ckeditor5-engine/docs/framework/guides/mini-inspector"
```

Comment:
Note to myself: remove this once mini inspector imported from the main inspector package.